subprojects/core-api/core-api.gradle.kts

```diff
@@ -38,7 +38,7 @@ dependencies {
 }
 
 gradlebuildJava {
-    moduleType = ModuleType.ENTRY_POINT
+    moduleType = ModuleType.WORKER
```

Comment:
We still handle this in the other PR, or have we given up making it Java 8?

Reply:
No given up, it's blocked on https://github.com/gradle/gradle/pull/9454